PORTAL/jobs/_utils.py

```diff
@@ -1704,7 +1712,7 @@ def ensure_local(self, reporoot=None, remotename=None):
             reporoot = os.path.join(HOME, f'{self.org}-{self.project}')
         repo = GitLocalRepo.ensure(reporoot, origin)
         if origin.name != 'origin':
-            repo.remotes.add(remote)
+            repo.remotes.add(remotename)
```

Comment:
We probably want the resolved name:

```suggestion
            repo.remotes.add(origin.name)
```